.github/workflows/deploy-fly-staging.yml

```diff
@@ -0,0 +1,18 @@
+name: Fly Deploy Staging
+on:
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+env:
+  FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+jobs:
+  deploy:
+      if: ${{ github.event.pull_request.merged }}
+      name: Deploy app to fly staging
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
+        - uses: superfly/flyctl-actions/setup-flyctl@master
+        - run: cp fly.toml.staging fly.toml
```

Comment:
where is `fly.toml.staging`?

Reply:
It is implemented in #35 where I introduced the prod config. fly.io looks for fly.toml but we will have two slightly different ones since there are two envs.

Reply:
[This](https://community.fly.io/t/how-to-run-seeds-exs-for-elixir-phoenix-app/3591/2) is how I ran seeds previously. Since we only need to do it once, this worked fine.